CMakeLists.txt

```diff
@@ -1,19 +1,12 @@
 cmake_minimum_required(VERSION 3.5)
 
 project(pplib
-    VERSION 1.0.5
     LANGUAGES C)
+set(VERSION 1.0.5)
 
 set(CMAKE_C_STANDARD 99)
 
-if(MSVC)
-    add_compile_options(/W4 /WX)
-else(MSVC)
-    add_compile_options(
-        -Wall -Wextra
-        -Wunused -Wimplicit -Wreturn-type -Wdeclaration-after-statement -Wno-unknown-pragmas -Wmissing-prototypes -Wmissing-declarations -Wparentheses -Wswitch -Wtrigraphs -Wcast-qual -Wcast-align -Wwrite-strings -Wold-style-definition -Wpointer-arith
-    )
-endif(MSVC)
+add_subdirectory(zlib)
```

Comment:
I get the following error if I try to run `make install` the project to a private prefix (that is: if I change `CMAKE_INSTALL_PREFIX`):
```
Install the project...
-- Install configuration: ""
-- Installing: /usr/local/lib/libz.1.2.11.dylib
CMake Error at zlib/cmake_install.cmake:44 (file):
  file INSTALL cannot copy file
  "/path/to/build/zlib/libz.1.2.11.dylib" to
  "/usr/local/lib/libz.1.2.11.dylib".
Call Stack (most recent call first):
  cmake_install.cmake:37 (include)
```

I actually wanted to test installation of `pplib` (without realizing this could have attempted installation of `zlib`).

Reply:
I haven't added `install` directives for `pplib` yet, so that wouldn't have worked in the first place.  However, you expressed that `zlib` should be taken from “the environment” (either system or built separately), so I will replace `add_subdirectory(zlib)` by `find_package(zlib)`.